ExtraCredit/DIIA/DerivedShapes/DerivedShapes.h

```diff
@@ -0,0 +1,18 @@
+
+#pragma once
+#include "Rectangle.h"
+#include "Shape.h"
+
+#ifdef DERIVEDSHAPESLIBRARY_EXPORTS
```

Comment:
```suggestion
#ifdef DERIVEDSHAPES_EXPORTS
```
You had a mismatch - in the .vcxproj you defined DERIVEDSHAPES_EXPORTS, not DERIVEDSHAPESLIBRARY_EXPORTS.